### PR TITLE
factoring out pfCHS into a new module

### PR DESCRIPTION
**Pull request by yiiyama**
I'm still not sure why you needed to recluster a full jet collection. Can we talk today whenever you are available?

### CODE REVIEW

File: Producer/python/utils/makeFatJets_cff.py [hunk 1]
```diff
@@ -88,13 +89,8 @@ def initFatJets(process, isData, labels):
...
 
-    # Charged hadron subtraction
-    addattr('pfCHS',
-        cms.EDFilter("CandPtrSelector",
-            src = cms.InputTag(pfSource),
-            cut = cms.string("fromPV")
-        )
-    )
+    # Add PF CHS
+    sequence += chsSequence(process, pfSource, skipIfExists = True)
```
Comment (dabercro) [inline Producer/python/utils/makeFatJets_cff.py:93]: Won't this not add pfCHS to the sequence since skipIfExists = True and the other bit is in prod.py?

Comment (yiiyama) [reply Producer/python/utils/makeFatJets_cff.py:93]: You are right. Updating the PR now

**Comment by mitsidekick** [comment]
Test comment

**Comment by dabercro** [comment]
Sorry, I meant to try that on my own PandaTree. Meet the Jenkins bot account.

**Pull request merged**